pandas/core/generic.pyi

```diff
@@ -19,7 +19,7 @@ _str = str
 class NDFrame(PandasObject, SelectionMixin, indexing.IndexingMixin):
     def __init__(self, data: BlockManager, axes: Optional[List[Index]]=..., copy: _bool=..., dtype: Optional[Dtype]=..., attrs: Optional[Mapping[Optional[Hashable], Any]]=..., fastpath: _bool=...) -> None: ...
     @property
-    def attrs(self) -> Dict[Optional[Hashable], Any]: ...
+    def attrs(self) -> Mapping[Optional[Hashable], Any]: ...
```

Comment:
I think that `Dict` is the correct thing here, as it's concrete and that's what the docs say.

Reply:
I'll change both the setter and getter to `Dict` then. They were previously mismatched.

Reply:
Hm. I wonder if it does actually accept mapping, or if it does just blindly set and get a dict. 

Apparently this is an experimental API anyway.